superduperdb/components/schema.py

```diff
@@ -40,7 +40,7 @@ def pre_create(self, db) -> None:
     @property
     def raw(self):
         return {
-            k: (v.identifier if not isinstance(v, Encoder) else 'Bytes')
+            k: (v.identifier if not isinstance(v, Encoder) else 'String')
```

Comment:
ok so we are moving away from bytes to string for Encoders type

Reply:
Yes

Reply:
> Is that efficient?

It will be a little less efficient than directly storing bytes

> Should this not be case-by-case?

Hey, @kartik4949 @blythed . I added a DBHelper to handle this. 
The default type is Bytes, which will be converted to String unless otherwise defined (such as clickhouse).

https://github.com/SuperDuperDB/superduperdb/pull/1496/commits/2d918faa7a9c02d8fe0866e60d0ee192d6deaffb